bin/restarter

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+MINIMUM_TIME_ALIVE = Integer(ENV.fetch("RESTART_MINIMUM_TIME_ALIVE", "3600"), 10)
+require "json"
+
+command = ARGV.dup.freeze
+start_time = Time.now
+
+iso_start = start_time.utc.strftime("%Y-%m-%dT%H:%M:%S%:z").freeze
+puts JSON.generate(restarter: {
+  start_time: iso_start,
+  startup: {
+    subprocess_args: command,
+    minimum_time_alive: MINIMUM_TIME_ALIVE
+
+  }
+})
+
+loop do
+  pid = Process.spawn(*command, out: :out, err: :err)
+  Process.wait(pid)
+  status = $?
+
+  puts JSON.generate(restarter: {
+    start_time: iso_start,
+    subprocess_exit: {
+      command: command,
+      exitstatus: status.exitstatus,
+      pid: status.pid,
+      success: status.success?
+    }
+  })
+
+  sleep(rand(1..10))
+
+  if Time.now - start_time > MINIMUM_TIME_ALIVE
+    puts JSON.generate(restarter: {start_time: iso_start,
+                                   shutdown: {start_time: iso_start}})
```

Comment:
Is there a reason you want the start_time logged in two separate places in the same message?

Reply:
oversight from refactoring, thank you